doc/api/crypto.md

```diff
@@ -3294,6 +3311,13 @@ the `crypto`, `tls`, and `https` modules and are generally specific to OpenSSL.
     <td><code>SSL_OP_PKCS1_CHECK_2</code></td>
     <td></td>
   </tr>
+  <tr>
+    <td><code>SSL_OP_PRIORITIZE_CHACHA</code></td>
+    <td>Instructs OpenSSL server to prioritize ChaCha20Poly1305
+    when client does.
+    This option has no effect if SSL_OP_CIPHER_SERVER_PREFERENCE
```

Comment:
Nit: Please wrap `SSL_OP_CIPHER_SERVER_PREFERENCE` in `<code>` or similar, too.

Reply:
Thanks! Fixed in following commit.